src/Integration/MefServices/VsSessionHost.cs

```diff
@@ -328,16 +327,6 @@ private void RegisterLocalServices()
 
                 return new ProjectSystemFilter(this, testProjectIndicator);
             }));
-            this.localServices.Add(typeof(IErrorListInfoBarController), new Lazy<ILocalService>(() =>
-                new ErrorListInfoBarController(this,
-                    new BindingChecker(
-                        new UnboundSolutionChecker(
-                            new ExclusionSettingsStorage(this.GetService<IConfigurationProviderService>(), Logger),
```

Comment:
Going forwards, `IConfigurationProvider` will only be available as a MEF service (i.e. when using the new Connected Mode settings/location).

However, I needed to do some more cleanup first.
This is part of it - dropping an unnecessary class that fetches `IConfigurationProviderService` as a service.

In the next PR I'll drop `IConfigurationProviderService` completely so that only the MEF-exported `IConfigurationProvider` is left.